bin/gltf-pipeline.js

```diff
@@ -49,7 +61,13 @@ var options = {
     binary : binary,
     embed : !separate,
     embedImage : !separateImage,
-    quantize : quantize
+    quantize : quantize,
+    ao_diffuse : ao_diffuse,
+    ao_scene : ao_scene,
+    ao_rayDepth : ao_rayDepth,
+    ao_resolution : ao_resolution,
+    ao_samples : ao_samples,
+    imageProcess : ao_diffuse
```

Comment:
`imageProcess` is needed at both the beginning and the end of the pipeline and will probably be needed in the pre-write stage, so I think it would be better to start using it now even though it's basically the same as the AO flag.